src/transport.ts

```diff
@@ -155,16 +155,12 @@ export class PinoSentryTransport {
     if (this.isSentryException(severity)) {
       const error = message instanceof Error ? message : new ExtendedError({ message, stack });
 
-      setImmediate(() => {
-        Sentry.captureException(error, scope);
-        cb();
-      });
+      Sentry.captureException(error, scope);
+      setImmediate(cb);
```

Comment:
those are not equivalent:

```ts
      setImmediate(() => {
        cb();
      });
```

```ts
      setImmediate(cb);
```

their `this` is different. not sure what the `cb` is and does it care about `this`.

Reply:
They code does `setImmediate(cb)` above, which is why I changed this to match.